status-im/status-react
# Hide send message button in group chat if user tries to spam

fixes #4474 

### Summary:

Hides send message button in group chat if user tries to spam.

Spam is considered spam when interval between current and last message sent by user is less than 1 second. When it happens, send button gets hidden for 2 seconds. If user spams again, cooldown gets increased to 5 seconds. Third cooldown gets raised to 10 seconds.

### Steps to test:
- Open Status
- Join public channel
- Send message
- Send one more message in less than 1 second
- Type something
- Check that send button is hidden



status: ready
